crates/scheduler/src/call/crypto/hash.rs

```diff
@@ -63,13 +67,18 @@ fn initialize<B: Board>(mut call: SchedulerCall<B, api::initialize::Sig>) {
     let result = try {
         let context = match convert_hash_algorithm::<B>(*algorithm)?? {
             #[cfg(feature = "board-api-crypto-sha256")]
-            Algorithm::Sha256 => HashContext::Sha256(board::crypto::Sha256::<B>::default()),
+            Algorithm::Sha256 => HashApi::<board::crypto::Sha256<B>>::new()
+                .map(|hash_api| HashContext::Sha256(hash_api.0)),
```

Comment:
I think we can change the definition of HashContext to make ShaX and HmacShaX take the wrappers. Which means here you should be able to write:
```suggestion
                .map(HashContext::Sha256),
```